contracts/interfaces/ICover.sol

```diff
@@ -242,10 +240,11 @@ interface ICover {
   error UnexpectedProductId();
 
   // Cover and payment assets
-  error PayoutAssetNotSupported();
-  error PaymentAssetDeprecated();
+  error CoverAssetNotSupported();
+  error InvalidPaymentAsset();
```

Comment:
Do we need to make changes in the UI code for these renames?

Reply:
I was in touch with the frontend team regarding custom errors. They'll also get back to us with a list of desired parameters for the errors.